### DISCUSSION TITLE
LiquidText alternative! but, why "workspace"?

### DISCUSSION THREAD

**Discussion by hmijail** [windingwind/zotero-better-notes#832] [Q&A] [answered]
Hi all, 

I just discovered zotero-better-notes, and I'm surprised because I looked for something like this for some time yet never heard of it before.

For some context, I was a subscriber to LiquidText, a PDF reader that is relatively famous in the Apple ecosystem. I dropped it because I found that Zotero works better than LT for my use cases, even though a few functionalities were missing. And it looks to me like ZBN actually fills the gaps, so this is great!

My problem is that the Readme file is not very clear, so I'm not sure if I'm understanding things correctly... So, please, consider improving the documentation :)

For starters, here's a question: 
When I tried to start for the first time ZBN it asks me to select a collection; it can't create the workspace note otherwise. And it looks like anyway any note can be opened as the workspace note. So, what is the difference between a workspace and a note? Or, why do we need the concept of the workspace at all?

**Comment by windingwind** [comment, marked as answer]
> When I tried to start for the first time ZBN it asks me to select a collection;

That's the behavior of a previous version of BN, which is indeed confusing. It's removed in the latest beta and you'll see a button to create/choose a `workspace` note only when you first open the workspace.

> why workspace

That's because of the *link current note to the `workspace`* feature. when you click the button at the top of the note editor, you don't need to choose which note to link to. Besides that, the `workspace` is just a tab where you can open a note with an outline and link preview.
